tests/evaluate/test_cross_validation.py

```diff
@@ -47,3 +61,50 @@ def test_leave_one_group_out_without_domain_adaptation(sample_data):
 def test_leave_one_group_out_with_domain_adaptation(sample_data):
     estimator = CoIRLS(kernel="linear", lambda_=1.0, alpha=1.0)
     check_leave_one_group_out_results(sample_data, estimator, domain_adaptation=True)
+
+
+@pytest.mark.parametrize("cv", [5, 10, LeaveOneGroupOut()])
+@pytest.mark.parametrize("estimator", [DummyClassifier(), LogisticRegression()])
+@pytest.mark.parametrize("transformer", [None, StandardScaler(), PCA()])
```

Comment:
Are `DummyClassifier` and `LogisticRegression` testing different functions? Similarly, are `StandardScaler` and `PCA` both necessary? Only keep one if not.